src/components/PlayWord/PlayWord.jsx

```diff
@@ -19,13 +21,25 @@ export class PlayWord extends React.Component {
     }
 
     sendMessage() {
-        this.props.playWord(this.state.currentValue);
+        axios.post('/api/validWord', {
```

Comment:
I think I left comments on a PR that got closed. I think this request should probably live in an action somewhere.